### PR TITLE
Convert parameters to static defaults

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -1,14 +1,10 @@
 # Certs Parameters
 class certs::params {
 
-  $pki_dir = '/etc/pki/katello'
```
Comment (ekohl) [inline manifests/params.pp:5]: This removal looks accidental

File: manifests/params.pp [hunk 2]
```diff
@@ -2,6 +2,7 @@
 class certs::params {
 
   $pki_dir = '/etc/pki/katello'
+  $nss_db_dir   = "${pki_dir}/nssdb"
```
Comment (ekohl) [inline manifests/params.pp:5]: This means it's no longer possible to change `$pki_dir` for `$nss_db_dir`, which it previously was. That's a regression.

**Review by ekohl — CHANGES_REQUESTED** [review]
Small nit about `$pki_dir` which you remove and later re-add. The `$nss_db_dir` change looks like a regression though. Overall :+1:

**Comment by ehelms** [comment]
Fixed up pki_dir and put back the nss_db setting.

**Review by ekohl — APPROVED** [review]

**Pull request merged**